app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,29 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#">Brand</a>
+    </div>
+
+    <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav navbar-right">
+        <% if user_signed_in? %>
+          <li><a href="#" class="navbar-link no-pointer">Welcome, <%= current_user.name %></a></li>
```

Comment:
Unless there is an existing feature that strictly depends on the users' names being saved to the db in a certain format, or you anticipate there being one soon, I'd recommend dealing with the casing at the presentation level.  i.e. using titleize in the views.

That said, if you'd like to format the names before being saved to the db, I'd recommend doing so by using a [callback](http://guides.rubyonrails.org/active_record_callbacks.html) in the `User` model.